spec/support/field_examples.rb

```diff
@@ -0,0 +1,33 @@
+module FieldExamples
```

Comment:
As field is quite a generic word you might want to call this DocumentTypeFieldExamples